_posts/2020-06-25-changelog-125.markdown

```diff
@@ -0,0 +1,176 @@
+* Support for `forbid` attribute (by [@Stzx])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5313 -->
+* Allow the user to change mutability of parameters in `Extract Function` dialog (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5548 -->
+* Report type errors for references with different mutability even these references contain unknown types (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5570 -->
+* Make `Self` inside impl of enum to be resolved to both impl and enum items (by [@fan-tom])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5580 -->
+* Enable `Convert match statement to if let` intention for irrefutable patterns (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5579 -->
+* Show unique editor tab names for `main.rs`, `lib.rs` and `Cargo.toml` (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5533 -->
+* Improve text for `Convert to type` quick fix (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5354 -->
+* Update `Wrong type arguments number` inspection (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5462 -->
+* Improve `Remove redundant type arguments` quick fix (by [@Stzx])
+
+## Fixes
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5599 -->
+* Fix false positive about not allowed `impl Trait` in associated type binding (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5381 -->
+* Fix usage replacement in the `Extract Enum Variant` intention (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5633 -->
+* Support tuple struct fields in `Make public` quick fix (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5547 -->
+* Don't trigger `Main function not found` inspection if the file has top-level errors (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5538 -->
+* Fix `Go to Declaration` from associated type bindings on 2020.2
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5574 -->
+* Fix parentheses completion for Fn* traits (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5528 -->
+* Liveness analysis takes into account destructured struct fields (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5523 -->
+* Fix `Join Lines` action for doc comment lines (by [@Stzx])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5515 -->
+* Disable `Promote Module to Directory` and `Downgrade Module to File` refactorings for invalid files in the editor (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/4868 -->
+* Correctly handle symlinked Cargo projects (by [@Jezza])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5603 -->
+* Take into account namespaces when name shadowing occurs
```

Comment:
Let's mentioned what it means for user, i.e. name resolution now works when user use `anyhow::anyhow` uses